semgrep-core/tests/python/concrete_syntax.py

```diff
@@ -1,11 +1,13 @@
 def foo():
```

Comment:
It's because to start the examples from ruby I copy-pasted a few one from python (which themselves were copy-pasted
from examples for javascripts) and realized some python examples were not as powerful as they could be (the javascript examples were better for instance).